proto/regen/ecocredit/v1/events.proto

```diff
@@ -259,3 +260,55 @@ message EventBurnRegen {
   // reason is the reason for the burn.
   string reason = 3;
 }
+
+// EventUpdateProjectClass is emitted when a project admin submits or updates
+// a project class relationship's metadata, usually relating to credit class
+// application.
+message EventUpdateProjectClass {
```

Comment:
@aaronc should we update these events to also use the new `ProjectEnrollment` terminology instead of `ProjectClass`?

Reply:
Oh thought I did. Updated now